src/main/config/secor.dev.hr.partition.properties

```diff
@@ -0,0 +1,11 @@
+include=secor.dev.properties
+
+secor.kafka.group=secor_hr_partition
+secor.message.parser.class=com.pinterest.secor.parser.ThriftMessageParser
+
+secor.s3.path=secor_dev/hr_partition
+secor.local.path=/tmp/secor_dev/message_logs/hr_partition
+
+message.timestamp.using.hour=true
```

Comment:
We should limit the number of params to bare minimum. Let's introduce a separate parser class for hourly partitions.